src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -343,6 +343,7 @@ The `navigate` method takes these arguments:
 		- `'push'`: the router will use `history.pushState` to create a new entry in the browser history. 
 		- `'replace'`: the router will use `history.replaceState` to update the URL without adding a new entry into navigation.
 		- `'auto'` (default): the router will attempt `history.pushState`, but if the URL is not one that can be transitioned to, the current URL will remain with no changes to the browser history.
+  - `formData`: A FormData object for `POST` requests.
```

Comment:
```suggestion
  - `formData`: A FormData object for `POST` and `GET` requests.
```
Should this be here, since it's mentioned below?

Reply:
Oh oops, I'll have to take this one back. `formData` is only for POST requests. For GET you just provide the URL with query params (which is what browsers do in `<form>` that are GET).